### ISSUE TITLE
[Urgent] Create form or input method for map viewers to report missing or bad data

### ISSUE THREAD

**Issue by daveluo** [covidcaremap/covid19-healthsystemcapacity#65]
We currently don't have an easy way for visitors and users of https://www.covidcaremap.org/maps/us-healthcare-system-capacity/#3.5/38/-96 to report, correct, or suggest adding data when they know or suspect an existing datapoint is wrong or missing. I.e.:
- a bed count may be outdated or wrong
- a facility name or location may be wrong
- a hospital which exists in real life is not in our dataset at all (e.g. https://github.com/covidcaremap/covid19-healthsystemcapacity/issues/49#issuecomment-603376587)
- new facilities are set up that are not in any existing records

We quickly need a simpler, faster way for anyone looking at our map data (probably through the visualization UI) to be able to notify us about what data row has an error, what data field(s) of that row, the nature of the error, and suggest a correction if available. This is primarily on the facility-level data because all of our aggregations are based off of this but users could ideally report at any level of our data.

The fastest, simplest way to create something for this could be a Google Form displayed as a link shown on the [map viz](https://www.covidcaremap.org/maps/us-healthcare-system-capacity/#3.5/38/-96) which:

1. lets any viewer enter data in the same fields as our dataset 
2. along with 2 more fields: one for a brief explanation of the suspected error and one for a data source if a correction is provided.
3. this populates a spreadsheet we monitor for new inputs and manually validate the suggested corrections for now

This spreadsheet could connect to or replace our error tracking thread and spreadsheet in #49.

**Comment by rcheetham** [comment]
DRAFT form is at: https://docs.google.com/forms/d/1wh97LGhbPS3XTA2wZckmKv5dGF4ZqnUxqsAaDipF-NE/edit?usp=sharing

Form so far has:

- Email address (*)
- What is the nature of your facility database update? (*) (multi-choice)
- What is the name of the facility in the COVID Care Map database? (*) (we aren't displaying any IDs, so we need to get the name)
- If this facility name is wrong, what the correct name?
- What is the city and state of the facility? (*) (Some facilities in different parts of the United States may share a name. We want to be sure we're correcting the right one.)
- What is the street address of the facility?
- How many staffed beds (all beds) in this facility?
- How many staffed ICU beds in this facility?
- What is the Bed Occupancy Rate (% of all beds)?
- What is the ICU Bed Occupancy Rate (%?of ICU Beds)?
- Briefly describe the correction, update, or addition you are submitting (*)
- What is your source for this data? (*)

Some questions:
- Are the fields set to required (noted w/ * above) the corrects ones?
- Wording changes?
- Should there be an email address to contact if they have problems with the form?

**Comment by daveluo** [comment]
Thanks @rcheetham this is looking great!

My take would be to move the email address to the bottom and make it optional so that doesn't get in the way of people wanting to submit reports of bad data even if they may not want to ID themselves for whatever reason. I would also make the "What is your source for this data? (*)" optional since they may not have a clear source. We'll be checking every data report for ourselves against the given source and other resources before updating it in our dataset. Open to your thoughts on this though.

So at the bottom would be:
- Briefly describe the correction, update, or addition you are submitting (*)
- What is your source for this data?
- Email address

Wording change:
"How many staffed beds (all beds) in this facility?" -> "How many staffed beds (all hospital beds of any type) in this facility?" 

Additional questions (optional):
- How many licensed beds (all beds) in this facility?
- How many licensed ICU beds in this facility?

RE: other questions:

> Should there be an email address to contact if they have problems with the form?

Good idea. I can set one up..how about data@covidcaremap.org or anything else you'd suggest?

> I can set up the spreadsheet with new records being sent to an email address. Do we want to do that?

Yea, good idea too, we could send them to ^ email address above? At least for now until that gets overwhelming..

> Who should have access to the spreadsheet?

I would keep it private for now and only accessible to the project maintainers (those w/ permissions to update the actual dataset). As we have people step up to be data verifiers, we can add them as viewers/editors too.

**Comment by rcheetham** [comment]
Thanks for the quick feedback @daveluo 

FWIW, I think source and email should be required - I think people should be able to describe how they know ("cuz I work there") and be willing to be contacted (anonymity invites bad behavior). However, I have changed them to match what you've suggested.

Added the additional questions and the email address to the intro paragraphs.

Where can I find a list of people with access to the actual dataset so that I can add them to an email rule?

**Comment by daveluo** [comment]
Thanks @rcheetham for the quick updates! 

I've modified a few questions and added a bit of branching logic to simplify things for user input.  Also made email/source mandatory as you suggested.

data@covidcaremap.org should be working now and I've added that to both the intro and post-submission acknowledgement. Also made the form results spreadsheet invite-only access for now.

This should do for now! Thanks again!
